src/main/java/com/sprint/mission/discodeit/dto/entity/BaseEntity.java

```diff
@@ -0,0 +1,24 @@
+package com.sprint.mission.discodeit.dto.entity;
+
+import lombok.Getter;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.util.UUID;
+
+@Getter
+public abstract class BaseEntity implements Serializable {
+   private final UUID id;
+   private final Instant createdAt;
+   private Instant updatedAt;
+
+    public BaseEntity() {
```

Comment:
패키지 내부에서만 사용하니 접근제어자를 protected로 바꿔도 될 것 같아요.  
```suggestion
    protected BaseEntity() {
```